docker/Dockerfile.dev

```diff
@@ -0,0 +1,25 @@
+FROM node:gallium-alpine
+
+ARG NAME_PROJECT=${NAME_PROJECT}
+ARG NAME_BRIDGE_DAPP=${NAME_BRIDGE_DAPP}
+ARG NAME_STATS_DAPP=${NAME_STATS_DAPP}
+ARG PORT_BRIDGE_DAPP=${PORT_BRIDGE_DAPP}
+ARG PORT_STATS_DAPP=${PORT_STATS_DAPP}
+
+ARG PATH_PROJECT=/app/${NAME_PROJECT}
+
+ENV PATH=${PATH_PROJECT}/node_modules/.bin:$PATH
+
+WORKDIR ${PATH_PROJECT}
+
+COPY package.json .
+
```

Comment:
Consider adding this line to install to improve the time to install the npm packages.

Reply:
i've pushed commit adding this [ae65b67](https://github.com/webb-tools/webb-dapp/pull/848/commits/ae65b67361c340f2075e97d631f94c71d0bfe86c)

without yarn.lock copied it took `time ./docker-dev.sh` duration `real	11m6.119s`
then with yarn.lock copied it took `time ./docker-dev.sh` duration `real	5m14.703s`

although it now shows this when building 
```
[1/4] Resolving packages...
warning Lockfile has incorrect entry for "@polkadot/api@9.2.4". Ignoring it.
warning Lockfile has incorrect entry for "@polkadot/keyring@10.1.7". Ignoring it.
warning Lockfile has incorrect entry for "@polkadot/networks@10.1.7". Ignoring it.
warning Lockfile has incorrect entry for "@polkadot/types@9.2.4". Ignoring it.
warning Lockfile has incorrect entry for "@polkadot/util@10.1.7". Ignoring it.
warning Lockfile has incorrect entry for "@polkadot/util-crypto@10.1.7". Ignoring it.
warning Lockfile has incorrect entry for "@polkadot/wasm-crypto@6.3.1". Ignoring it.
warning Lockfile has incorrect entry for "typescript@~4.8.2". Ignoring it.
```